sys/shell/commands/sc_sys.c

```diff
@@ -29,3 +30,18 @@ int _reboot_handler(int argc, char **argv)
 
     return 0;
 }
+
+int _echo_handler(int argc, char **argv)
```

Comment:
This belongs into its own PR.

Reply:
Also, I noticed today, that this somewhat clashes with [a task in our tutorials](https://github.com/RIOT-OS/Tutorials/blob/master/task-02/README.md#task-21-a-simple-echo-command-handler)